dueutil/botcommands/weapon.py

```
@@ -632,3 +637,142 @@ def weapon_info(weapon_name=None, **details):
+    """
+
+    extras = {"melee": not ranged, "icon": icon}
+    if image_url is not None:
+        extras["image_url"] = image_url
+
+    if "image_url" in extras and not await imagehelper.is_url_image(image_url):
+        extras.pop("image_url")
+        await imagehelper.warn_on_invalid_image(ctx.channel)
+
+    weapon = weapons.GlobalWeapon(name, hit_message, damage, accy, **extras, ctx=ctx)
+    dbconn.insert_global_weapon(weapon.id, weapon)
```

Ruff

Undefined name `dbconn`

(F821)